internal/eventstore/handler/v2/statement.go

```diff
@@ -504,6 +504,16 @@ func NewJSONCol(name string, value interface{}) Column {
 	return NewCol(name, marshalled)
 }
 
+func NewIncrementCol(column string, value interface{}) Column {
```

Comment:
I would start introducing constraints in this package. For value it would be `constraints.Integer`
```suggestion
func NewIncrementCol[Int constraints.Integer](column string, value Int) Column {
```